.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -155,6 +155,11 @@ else
         fi
         pushd ../
         make tpgtools OUTPUT_PATH=$LOCAL_PATH VERSION=$VERSION
+
+        # Only generate TeamCity-related file for TPG and TPGB
+        if [ "$VERSION" == "ga" ] || [ "$VERSION" == "beta" ]; then
```

Comment:
Should we test this by adding an `if [ "$PR_NUMBER" == N ] ` condition?

Reply:
Do we want to test it in this PR?  If so, we'll have to build a image locally, upload it with a tag and use that in this PR and we'll also have to whitelist the PR in the evil-build-killer. Let me know if that's needed, I can go ahead and make the change

Reply:
Thanks, Shuya. I will test the image in another PR.

Reply:
If the changes in [ddc1d43](https://github.com/GoogleCloudPlatform/magic-modules/pull/8344/commits/ddc1d43343aa34b3a7ff0b4ae77cad20e0166e6c) look OK to you then I'm happy to not test this on a special PR.

Reply:
I don't think we can test it in other PRs (?), as we need the change in this PR to be able to generated TeamCity-related file

Reply:
Test the tool locally and it worked. But technically adding this CI change will break all current PRs. They all need to be rebased to get the tool in, otherwise will run into `No rule to make target 'teamcity-servicemap-generate'` error

Reply:
I don't know how to avoid that 😬